llvm/test/Transforms/LoopVectorize/RISCV/pr88802.ll

```diff
@@ -1,7 +1,57 @@
-; REQUIRES: asserts
-; RUN: not --crash opt -passes=loop-vectorize -mtriple=riscv64 -mattr=+v -disable-output %s
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
+; RUN: opt -passes=loop-vectorize,dce,simplifycfg -mtriple=riscv64 -mattr=+v -S %s | FileCheck %s
```

Comment:
We usually try to avoid having the tests depend on other passes, to avoid them breaking if other passes change.